### PR TITLE
small changes to Color pages

### PR DESCRIPTION
**Pull request by Jay-o-Way**
## Description
Changed one "control" to "card", make every Description end with a period.

## Motivation and Context
just tweaking

## How Has This Been Tested?
With Visual Studio :)

## Screenshots (if appropriate):
image

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by Jay-o-Way** [comment]
Question:
Schermafbeelding 2024-11-28 174351

**Comment by Zakariathr22** [comment]
> Question: Schermafbeelding 2024-11-28 174351

I believe this is another issue that occurs when the theme is toggled and should be reported, because the background of the "Smoke Background" color example does not update when the theme is toggled.
**Before toggling the theme**
image
**After toggling the theme** 
image

File: WinUIGallery/Controls/DesignGuidance/ColorSections/BackgroundSection.xaml [hunk 1]
```diff
@@ -77,7 +77,7 @@
             <designguidance:ColorTile
                 Background="{ThemeResource SmokeFillColorDefaultBrush}"
                 ColorBrushName="SmokeFillColorDefaultBrush"
-                ColorExplanation="Dims backgrounds behind dialogs"
+                ColorExplanation="Dims background behind dialogs"
```
Comment (niels9001) [inline WinUIGallery/Controls/DesignGuidance/ColorSections/BackgroundSection.xaml:80]:
```suggestion
                ColorExplanation="Dims the background behind dialogs"
```

File: WinUIGallery/Controls/DesignGuidance/ColorSections/StrokeSection.xaml [hunk 2]
```diff
@@ -288,7 +288,7 @@
         <designguidance:ColorPageExample
             Title="Focus Stroke"
             Background="{ThemeResource SolidBackgroundFillColorQuarternaryBrush}"
-            Description="Used for divider and graphic lines. Theme inverse; dark in light theme and light in dark theme..">
+            Description="Used for divider and graphic lines. Theme inverse; dark in light theme and light in dark theme.">
```
Comment (niels9001) [inline WinUIGallery/Controls/DesignGuidance/ColorSections/StrokeSection.xaml:291]:
```suggestion
            Description="Used for divider and graphic lines.">
```

If we remove it for `Divider Stroke` should we remove it here as well? Or leave them in for both?

**Review by niels9001 — CHANGES_REQUESTED** [review]

File: WinUIGallery/Controls/DesignGuidance/ColorSections/StrokeSection.xaml (hunk 2 above)
Comment (Jay-o-Way) [reply WinUIGallery/Controls/DesignGuidance/ColorSections/StrokeSection.xaml:291]: Nope. Here, the claim is **invalid**: the color is lighter in the light theme. The other one is correct.

Comment (niels9001) [reply WinUIGallery/Controls/DesignGuidance/ColorSections/StrokeSection.xaml:291]: Ah yes, you're right! Nvm

**Review by marcelwgn — APPROVED** [review]

**Pull request merged**